src/Cache/CacheOperationTypes/CacheOperationTypes.php

```diff
@@ -459,6 +459,84 @@ class CacheDeleteResponseError extends CacheDeleteResponse
     use ErrorBody;
 }
 
+abstract class CacheKeysExistResponse extends ResponseBase
+{
+    public function asSuccess() : CacheKeysExistResponseSuccess|null
+    {
+        if ($this->isSuccess()) {
+            return $this;
+        }
+        return null;
+    }
+
+    public function asError() : CacheKeysExistResponseError|null
+    {
+        if ($this->isError()) {
+            return $this;
+        }
+        return null;
+    }
+}
+
+class CacheKeysExistResponseSuccess extends CacheKeysExistResponse
+{
+    private array $values = [];
+
+    public function __construct(_KeysExistResponse $response) {
+        parent::__construct();
+        foreach ($response->getExists()->getIterator() as $value) {
+            $this->values[] = (bool)$value;
+        }
+    }
+
+    public function exists() : array {
```

Comment:
nit, up to you: i am imagining that for most of the SDKs we will want to provide a dictionary accessor for this.  If we do it as an array like this, it's useful if the caller wants to do a `.every` or `.all` check, but otherwise they have to loop over this and their original fields list in parallel to check values and it's kinda janky. 

I feel like as an end user it's a much nicer experience if I can get back a dictionary with the fields as keys.

we can always add this later, so definitely not a blocker.

Reply:
I'm fine with that, but I was going off [the SDK API spec](https://www.notion.so/momentohq/SDK-API-Methods-Spec-cf12e4db2535405ca2acc3608814b083#fbd08eaf1b614a579728f38263dd13df) and it explicitly calls for a list of booleans. For that reason, the golang SDK is returning []bool for this too. I totally agree that a dictionary is a better UX choice. I'm happy to change this up to return a dictionary (which because this is PHP will still be called and typed "an array") but we should think about adding `existsMap` or similar to the gloang SDK as well. And we should obviously correct the spec.

Reply:
we need the list of booleans.  but i think adding a map too is nice.  you can see how we did it in the .NET SDK if you like.  And yes if you wouldn't mind updating the spec that would be great, thank you!!

Reply:
and adding it to go at some point would be great too but it's an additive change so we can do it whenever IMO

Reply:
Added a "dictionary" accessor in 1a803f98 and updated the spec to reference `existsDictionaryString` and `existsDictionaryBytes` accessors. @cprice404 I opted to omit a "Bool" suffix on those accessors but feel free to change that if you prefer the more verbose version. Because PHP doesn't differentiate, I just added the accessor in this SDK as `existsDictionary`.